staff/manuelbarzi/cv/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Manu Wave - CV</title>
+</head>
+<body>
+    <h1>Manu Wave - CV</h1>
+
+    <p>I think I'm a good person, but sometimes I get angry too easily, but... well, it doesn't matter, does it?</p>
+
+    <h2>Position</h2>
+
+    <h3>Trainer at SkyLab Coders Academy</h3>
+
+    <h2>Past</h2>
+
+    <h3>Java &amp; Javascript Enginner at WTF</h3>
+
+    <h4>Java &amp; Javascript Enginner at Ortivamedia</h4>
+
+    <h3>Java &amp; Javascript Enginner at OMG</h3>
+
+    <h4>Java &amp; Javascript Enginner at Super Chachi Co</h4>
```

Comment:
this and line #22 are <h4> but lines #20 & #24 are <h3>? Aren't they all in the same section, at the same level?

Reply:
+1 @Melinx

Reply:
@Melinx they are not at the same level because those where positions at sub-contracted companies related to the previous.

Reply:
That's not clear... maybe the companies are not going to understand this, i think you can specify it better.